examples/serverless-agent/fargate/workload/providers.tf

```diff
@@ -0,0 +1,4 @@
+provider "aws" {
+  region = var.region
+  profile = var.profile
+}
```

Comment:
Shouldn't we add the sysdig provider here?

Reply:
oh, nvm, it's in versions.tf

Reply:
Happy to combine / rearrange. There's a dozen ways to lay out these projects.